build/scripts/entrypoint-volume.sh

```diff
@@ -32,5 +32,9 @@ fi
 # into a persistent volume
 export VSCODE_AGENT_FOLDER=/checode/remote
 
+if [ -z "$CODE_HOST" ]; then
+  CODE_HOST="127.0.0.1"
+fi
+
 # Launch che without connection-token, security is managed by Che
-./node out/server-main.js --port 3100 --without-connection-token
+./node out/server-main.js --host ${CODE_HOST} --port 3100 --without-connection-token
```

Comment:
probably it requires quotes around the env var but not required to merge this PR

Reply:
What problems could this potentially cause?
I think the quotes are redundant here.

Reply:
@azatsarynnyy https://github.com/koalaman/shellcheck/wiki/SC2086

<img width="820" alt="image" src="https://user-images.githubusercontent.com/606959/165247858-eb1d9dc4-3ed5-4fc3-a2c4-30a404d8d00e.png">

Reply:
Thank you @l0rd for pointing that out.
Fixed in https://github.com/che-incubator/che-code/pull/42